README.md

```diff
@@ -1,40 +1,75 @@
 # Windows Machine Config Operator
 
 ## Introduction
-The Windows Machine Config Operator configures Windows Machines into nodes, enabling Windows container workloads to
-be ran within OKD/OCP clusters. The operator is configured to watch for [Machines](https://docs.openshift.com/container-platform/4.5/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws) with a `machine.openshift.io/os-id: Windows`
-label. The way a user will initiate the process is by creating a MachineSet which uses a Windows image with the Docker
-container runtime installed. The operator will do all the necessary steps to configure the underlying VM so that it can join
-the cluster as a worker node.
+The Windows Machine Config Operator configures Windows instances into nodes, enabling Windows container workloads to
```

Comment:
Is instance an all encompassing word for instances and machines? Should we say `configures instances and machines into nodes`

Reply:
The terminology around this is pretty confusing, but its important to differentiate "Machines" and the actual VM that gets created as a result of the Machine.

A Machine is a kubernetes object that defines the desired state of virtual machines available to the cluster. Technically a Machine isnt something that can be configured, instead you would configure the VM associated with it. 

Used in this context, instance would be a cover-all term for VMs and other potential configuration targets, possibly including a bare metal host.

Reply:
Thanks for the context! Would it make sense to someday change our name to Windows Instance Config Operator (WICO)? :smiley:

Reply:
That name would make sense, but it would be logistically hard to switch at this point 😅